backend/src/routes/auth.js

```diff
@@ -54,7 +55,9 @@ router.post('/signup', async (req, res) => {
 
   let userType = req.body.userType
   if (!userType) userType = 'customer'
-
+  const salt = await bcrypt.genSalt(10)
```

Comment:
@aashutosh148 this code is repeated could we extract it over to a different file in `/utils/`, so that it could be easily reused in other projects as well.

Reply:
Sure thing,
I have moved it under utils and made the necessary changes.